### DISCUSSION TITLE
ERROR:The path to the model does not exist. Exiting.

### DISCUSSION THREAD

**Discussion by membrath** [oobabooga/text-generation-webui#2467] [Q&A]
I keep getting the following error when I try to start server.py with the following command - python server.py --model llama-13b-4bit

C: Users\me\miniconda3\envs\llama4bit2\lib\site-packages\bitsandbytes\libbitsandbytes_cuda117.dll
INFO:Loading llama-13b-4bit...
ERROR:The path to the model does not exist. Exiting.
Running on local URL:  http://127.0.0.1:7860

Can anyone help.....where do I need to add the 'path' to my models.

**Comment by BetaDoggo** [comment]
Models should be placed in the models directory within their own folder.
ex: ./models/llama-13b-4bit/(models files here)
If it's layed out that way the command you're using should work.
